### PR TITLE
:book: Add missing Travis badge

### PR DESCRIPTION
**Pull request by neilbmclaughlin**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-58.dev.c2s.nhschoices.net](http://gp-finder-pr-58.dev.c2s.nhschoices.net)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # GP Finder
 
+
```
Comment (st3v3nhunt) [inline README.md]: Do these 2 additional blank lines need to be here?

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-58.dev.c2s.nhschoices.net](http://gp-finder-pr-58.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-58.dev.c2s.nhschoices.net](http://gp-finder-pr-58.dev.c2s.nhschoices.net)

**Comment by neilbmclaughlin** [comment]
@st3v3nhunt - good for re-review.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
 # GP Finder
 
+[![Build Status](https://travis-ci.org/nhsuk/gp-finder.svg?branch=master)](https://travis-ci.org/nhsuk/gp-finder)
 [![Coverage Status](https://coveralls.io/repos/github/nhsuk/gp-finder/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/gp-finder?branch=master)
+[![Known Vulnerabilities](https://snyk.io/test/github/nhsuk/gp-finder/badge.svg)](https://snyk.io/test/github/nhsuk/gp-finder)
 
-A GP finder application. The results will be pointing to the profile app for booking an appointment with a GP or register with a GP.
-
+A GP surgery finder application to allow the user to easily navigate to a GP surgeries online booking system.
```
Comment (st3v3nhunt) [inline README.md]: If I was being ultra picky I might say `GP surgeries` should be `GP surgery's`

**Pull request merged**